easybuild/easyconfigs/g/GraphMap2/GraphMap2-0.6.4-foss-2019b.eb

```diff
@@ -39,16 +40,21 @@ checksums = [
 files_to_copy = [(['bin/Linux-x64/graphmap2'], 'bin')]
 
 # Move sources dependencies to expected folders
-prebuildopts = "mv %(builddir)s/seqlib-*/*  %(builddir)s/%(name)s-%(version)s/codebase/seqlib/ && " \
-               "mv %(builddir)s/argparser-*/*  %(builddir)s/%(name)s-%(version)s/codebase/argumentparser/ && " \
-               "mv %(builddir)s/gindex-*/*  %(builddir)s/%(name)s-%(version)s/codebase/gindex/ && "
+prebuildopts = "mv %(builddir)s/seqlib-*/* %(builddir)s/%(namelower)s-%(version)s/codebase/seqlib/ && "
+prebuildopts += "mv %(builddir)s/argparser-*/* %(builddir)s/%(namelower)s-%(version)s/codebase/argumentparser/ && "
+prebuildopts += "mv %(builddir)s/gindex-*/* %(builddir)s/%(namelower)s-%(version)s/codebase/gindex/ && "
 
-prebuildopts += 'export GCC="$CXX" && '
+buildopts = 'GCC="$CXX" '
+buildopts += 'CC_FLAGS_RELEASE="$CXX_FLAGS -DRELEASE_VERSION -c -g -m64 '
```

Comment:
@Darkless012 Most important remark: change `$CXX_FLAGS` to `$CXXFLAGS`. Only the latter is set by EasyBuild, so when using `$CXX_FLAGS` you're effectively compiling without any optimization flags... 😱 

I would also drop the `-g` (that's for enabling debug symbols. and controllable via `debug` in `toolchainopts`), and also `-m64` (that's the default in modern compilers).